include/crow/routing.h

```diff
@@ -694,13 +708,16 @@ namespace crow
 
     const int RULE_SPECIAL_REDIRECT_SLASH = 1;
 
+
     /// A search tree.
     class Trie
     {
     public:
         struct Node
         {
-            unsigned rule_index{};
+            uint16_t rule_index{};
+            // Assign the index to the maximum 32 unsigned integer value by default so that any other number (specifically 0) is a valid BP id.
+            uint16_t blueprint_index{0xFFFF};
```

Comment:
`constexpr uint16_t INVALID_BP_ID = 0xFFFF`?

Reply:
definitely something to delete, my bad..

Reply:
I'm a little confused, if the value is never going to change, why not use `const`?